concordium-consensus/src/Concordium/External.hs

```diff
@@ -891,14 +891,17 @@ decodeBlockHash blockcstr = readMaybe <$> peekCString blockcstr
 decodeAccountAddress :: CString -> IO (Either String AccountAddress)
 decodeAccountAddress acctstr = addressFromBytes <$> BS.packCString acctstr
 
--- |Decode a null-terminated string as either an account address (base-58) or a
+-- |Decode a null-terminated string as either an account address (base-58), account index (Int) or a
 -- credential registration ID (base-16).
-decodeAccountAddressOrCredId :: CString -> IO (Maybe (Either CredentialRegistrationID AccountAddress))
+decodeAccountAddressOrCredId :: CString -> IO (Maybe AccountIdentifier)
```

Comment:
What is "AccountIdentifier"?

Reply:
It is defined in `consensus-base`, forgot to push it. coming...

Reply:
This should probably be renamed `decodeAccountIdentifier`.

Reply:
done 👍